src/services/config.js

```diff
@@ -8,7 +8,13 @@ config.parseServerConfig()
 config.parseDatabaseConfig()
 config.parseKeyConfig()
 
-if (process.env.NODE_ENV === 'unit') {
+function isRunningUnitTests () {
```

Comment:
Yeah, "unit" is somewhat vague, but it's probably better to drop it to avoid potential confusion.